src/networking.c

```diff
@@ -1538,6 +1540,11 @@ void freeClient(client *c) {
     /* Notify module system that this client auth status changed. */
     moduleNotifyUserChanged(c);
 
+    /* Free the RedisModuleBlockedClient held onto for reprocessing if not already freed. */
+    if (c->module_blocked_client) {
+        zfree(c->module_blocked_client);
+    }
```

Comment:
```suggestion
    zfree(c->module_blocked_client);
```
zfree() internally does a null check.

Reply:
Thanks - I removed the null check here: https://github.com/redis/redis/pull/11659/commits/2e5c117eafed8fcf20ddad649fbaf3bbdfd3edbc